geode-core/src/main/java/org/apache/geode/internal/security/SecurityService.java

```diff
@@ -71,4 +71,7 @@ Object postProcess(Object principal, String regionPath, Object key, Object value
   SecurityManager getSecurityManager();
 
   PostProcessor getPostProcessor();
+
+  void logoutJmxUser(String user, Subject subject);
```

Comment:
This will break backward compatibility and, considering that this interface is the entry point for users to customize security, we certainly don'y want to do that. You can try to use [default methods](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html) instead, but double check anyway that API backward compatibility is not broken.

Reply:
The method `logoutJmxUser` is being added to `SecurityService` which is an internal interface. This shouldn't break backward compatibility. The two security related interfaces that are exposed to users and would break backward compatibility if changed are `SecurityManager` and `PostProcessor`. Neither of those interfaces are being changed in this PR, so this change should be ok.

Reply:
Good point @kirklund, thanks for correcting my comment!.